packages/garbo/src/outfit/dropsgear.ts

```diff
@@ -364,3 +365,22 @@ function powerGlove(): Map<Item, number> {
     ],
   ]);
 }
+
+const POSSIBLE_SNEEGLEEB_DROPS = Item.all().filter(
+  (i) =>
+    i.tradeable && i.discardable && (i.inebriety || i.fullness || i.potion),
+);
+function sneegleebs(): Map<Item, number> {
+  const dropValue =
```

Comment:
May be worth caching this value, it's currently over 2000 items being averaged every time and gets checked every combat.